butterfree/clients/cassandra_client.py

```diff
@@ -42,7 +42,7 @@ class CassandraClient(AbstractClient):
     def __init__(
         self,
         host: List[str],
-        keyspace: str,
+        keyspace: str = None,
```

Comment:
is it ok to set up a connection without a defined Keyspace? Line 79 of this file runs a `cluster.connect(self.keyspace)`, are we ok with it being `None`?

Reply:
From my perspective, I think ```keyspace``` should be a required argument as well, therefore I've changed this.